src/Features/Core/Portable/Completion/Providers/AbstractInternalsVisibleToCompletionProvider.cs

```diff
@@ -227,6 +238,26 @@ private async Task<string> GetAssemblyNameFromInternalsVisibleToAttributeAsync(D
             return string.Empty;
         }
 
+        private static async Task<TextSpan> GetTextChangeSpanAsync(Document document, TextSpan startSpan, CancellationToken cancellationToken)
+        {
+            var result = startSpan;
+            var syntaxFacts = document.GetLanguageService<ISyntaxFactsService>();
+            var root = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
+            var token = root.FindToken(result.Start);
+            if (syntaxFacts.IsStringLiteral(token) || syntaxFacts.IsVerbatimStringLiteral(token))
+            {
+                var text = root.GetText();
+                
+                // Expand selection in both directions until a double quote or any line break character is reached
+                bool IsWordCharacter(char ch) => !(ch == '"' || (char.IsControl(ch) || char.IsSeparator(ch) && !char.IsWhiteSpace(ch)));
```

Comment:
1. this is super hard to read.
2. it doesn't match your comment.
3. why not actually just check for double-quote/line-break?  why the need for Control/Separator/Whitespace??

Reply:
I wanted to check for the unicode categories LineSeparator and ParagraphSeparator but.netstandard 1.3 doesn't provide a unicode category method. So I used the IsSeparator method and fixed what was missing and what was too much. I will change it to explicit call out the line break characters instead.

I was also thinking about using your IVirtualCharService to get the bounds of the text inside the StringLiteralToken. Then I only need to make sure that the span stays on the same line (It seems at least in VB the completion fails if the CompletionSpan spans more then one line). What do you think about that approach?

Reply:
I think it's overkill :).  We know what 99.9999% of all IntervalsVisibleTo strings will look like.  It seems like it should just be sufficient to eat forward/backward until you hit one of:

1. quote
2. end-of-line
3. end-of-file